### PR TITLE
Update local build instructions

### PR DESCRIPTION
**Pull request by moonchen**
Update README.md with local build prerequisites.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,16 @@ VAGRANT_PREFER_SYSTEM_BIN=1 vagrant ssh -c "sudo maestro"
 
 ### Local Build
 
+You can also build without using Vagrant.
+
+#### Prerequisites
+
+* Linux
+* go (tested on go 1.15.1)
+* libuv
```
Comment (costanic) [inline README.md:116]: should be: libuv1-dev

I also noticed libunwind-dev listed as a dependency in the distro-pelion-edge recipe.  Is that needed here?

**Review by costanic — CHANGES_REQUESTED** [review]
`apt install libuv1-dev` is what worked for me.

File: README.md (hunk 1 above)
Comment (moonchen) [reply README.md]: libunwind was a dependency for libgrease.  It's no longer needed.

Comment (moonchen) [reply README.md]: > should be: libuv1-dev

OK.  Added as a hint for Debian users.

**Review by costanic — APPROVED** [review]

**Pull request merged**